calculator.js

```diff
@@ -5,17 +5,67 @@ const exampleAdditionInput = {
   operation: 'add',
 }
 
+const isInteger = function(input) {
+  if (isNaN(input)) console.log(`"${input}" is not a number`);
+  return parseInt(input);
```

Comment:
oops you forgot your curly braces, and I think an `else` statement would help here; otherwise, if the input is `NaN`, it will still try and `parseInt` the input instead of breaking out of the program.

Try turning lines 9 and 10 into an if/else statement